### PR TITLE
Pre annotation 

### LINKED ISSUE #105: Documents with pre-annotations
Allow project manager to specify field to look for pre-annotation

* [x] Add pre-annotation field in project config, blank by default (no pre-annotation)
* [x] Add feature where AnnotationRenderer can use a default set of label values

### PR DESCRIPTION
**Pull request by twinkarma**
Resolves #105 

* Added `document_pre_annotation_field` to Project model
* `Project.get_annotation_task_dict` now adds the `document_pre_annotation_field` value
* [UI] Added field in project configuration for setting the pre annotation field
* [UI] Updated AnnotationRenderer to lookup document for pre-annotation
* [UI] Updated `ProjectConfiguration.vue` and `Annotate.vue` to pass pre-annotation field to `AnnotationRenderer`
* Added unit test to AnnotationRenderer for pre-annotation
* Add integration test for pre-annotation
* Added user documentation on pre-annotation

### CODE REVIEW

File: frontend/src/components/ProjectConfiguration.vue [hunk 1]
```diff
@@ -101,14 +101,15 @@
         </b-form-checkbox>
       </b-form-group>
       <b-form-group label="Auto elevate to annotator"
-        description="Automatically allows annotation on real dataset after the training and testing stage. A disabled stage counts as having been completed.">
+                    description="Automatically allows annotation on real dataset after the training and testing stage. A disabled stage counts as having been completed.">
```
Comment (davidwilby) [inline frontend/src/components/ProjectConfiguration.vue:104]: ?

File: package.json [hunk 2]
```diff
@@ -7,6 +7,8 @@
     "postinstall": "(cd frontend && npm install --unsafe-perm) #Install npm packages in frontend also",
     "migrate": "./manage.py migrate",
     "migrate:integration": "./migrate-integration.sh",
+    "db:create": "createdb -p 5432 -O user teamware_db",
```
Comment (davidwilby) [inline package.json:10]: The db name is defined in .env, is it possible to get it from there in the package scripts?

**Review by davidwilby — APPROVED** [review]
LGTM, is there anything that needs to be added to the documentation for this feature?

File: frontend/src/components/ProjectConfiguration.vue (hunk 1 above)
Comment (twinkarma) [reply frontend/src/components/ProjectConfiguration.vue:104]: The description attribute has just been aligned with the label attribute. Or is there something wrong with the description?

Comment (davidwilby) [reply frontend/src/components/ProjectConfiguration.vue:104]: Nope nothing wrong, sorry I was just wondering why the indent, but makes sense that it's for alignment.

File: package.json (hunk 2 above)
Comment (twinkarma) [reply package.json]: I'd just put it in for testing but it's probably best to leave it out so I've removed it with the latest commit

Comment (twinkarma) [reply package.json]: But I did find a tutorial on how to add .env file into package.json scripts though https://www.genui.com/resources/env-variables-json

**Pull request merged**